uniflow-core/src/main/kotlin/io/uniflow/core/flow/UIDataStore.kt

```diff
@@ -11,14 +11,15 @@ class UIDataStore(private val publisher: UIDataPublisher, defaultState: UIState)
         private set
 
     suspend fun pushNewData(dataUpdate: UIDataUpdate) {
+        UniFlowLogger.debug("UIDataStore - push: $dataUpdate")
         when (dataUpdate.data) {
             is UIState -> {
                 currentState = dataUpdate.data
                 UniFlowLogger.logState(dataUpdate.data)
                 when (dataUpdate.type) {
                     UIDataUpdateType.PUBLISH -> publisher.publishState(dataUpdate.data)
                     else -> {
-                        UniFlowLogger.log("state notification - ${dataUpdate.data} is not published")
+                        UniFlowLogger.log("[State] internal update '${dataUpdate.data}'")
```

Comment:
Should you use `UniFlowLogger.debug` instead of a regular log?

Reply:
yep, still thinking about that 🤔  I believe it's useful to know the notify thing ... but has to be clear that also that we don't push the state, just the event. But logs are explicit enough